src/work-allocation/components/task-manager-filter/task-manager-filter.component.ts

```diff
@@ -87,11 +91,12 @@ export class TaskManagerFilterComponent implements OnChanges {
     this.sessionStorageService.setItem(FilterConstants.Session.TaskManager, toStore);
   }
 
-  public getLocationById(id: string): Location {
+  public getLocationById(id: any): Location {
```

Comment:
Should never use any. Either make changes in the rest of the code so it's always a string, or, if that can't be done, make this a compound type.

Reply:
Agreed, changed it to string